source/home.html

```diff
@@ -25,7 +25,7 @@
           <a class="product-list-item {{ theme.show_overlay }}" href="{{ product.url }}" title="View {{ product.name | escape }}">
             <div class="product-list-item-container">
               <figure class="product-list-image-container">
-                <img alt="Image of {{ product.name | escape }}" class="fade-in product-list-image {% if product.image.height > product.image.width %}image-tall{% elsif product.image.height < product.image.width %}image-wide{% else %}image-square{% endif %}" src="{{ product.image | product_image_url | constrain: 1000, 1000 }}">
+                <img alt="View product" class="fade-in product-list-image {% if product.image.height > product.image.width %}image-tall{% elsif product.image.height < product.image.width %}image-wide{% else %}image-square{% endif %}" src="{{ product.image | product_image_url | constrain: 1000, 1000 }}">

```

Comment:
I'm curious what you all think of this approach. When using the screen reader, I noticed that it was playing these words:

<product name> <product status><product name><product price>

So product name was in there twice. I realized it's because the link contains many different elements, including the info that appears on hover/focus. 

I put some more info in the commit message as well as a link to some of the resources I looked at to make this choice. The screen reader now says (screenshot from Mac VoiceOver):

<img width="755" alt="Screen Shot 2019-11-06 at 1 38 57 PM" src="https://user-images.githubusercontent.com/13512356/68328081-98ff7c80-009c-11ea-806e-95777724ffad.png">

Reply:
hmmm, my thought is that we keep it tied to the actual element, in this case, since it is an image, "View product" isn't really describing what that specific element is to me. If "image of <product name> is too redundant, perhaps call it simply using "product image?"

Reply:
Ah but I just read the resource you've chosen, which makes sense to me now, so I'm good to keep what you've got.